Tests/pens/ftPen_test.py

```diff
@@ -0,0 +1,115 @@
+        pen = FTPen(None)
+        draw_cubic(pen)
+        offset, width, height = (0, 0), 500, 500
+        buf1, _ = pen.buffer(offset=offset, width=width, height=height)
+        buf2 = zlib.decompress(base64.b64decode(ZLIB_B64_BIN))
+        self.assertEqual(buf1, buf2)
```

Comment:
asserting that the rendered buffer is exactly equal to something may be too precise for our tests, as it could fail unless a specific freetype version is used. Maybe we could relax this a bit, just assert that it is not empty or use Pillow to diff images within a threshold